conf/fedora2-native.xml

```diff
@@ -66,13 +70,9 @@
     <bean id="namespacePrefixMapper" class="org.fcrepo.migration.foxml.NamespacePrefixMapper">
         <constructor-arg name="namespaceFile" type="java.io.File" ref="namespaceFile"/>
     </bean>
-
-    <bean id="fedora4Client" class="org.fcrepo.migration.f4clients.DefaultFedora4Client">
-        <constructor-arg name="repo" ref="repo" />
-    </bean>
-
-    <bean id="repo" class="org.fcrepo.client.impl.FedoraRepositoryImpl">
-        <constructor-arg name="repositoryURL" ref="fedora4Url" />
+    
+    <bean id="fedora4Client" class="org.fcrepo.migration.f4clients.StatelessFedora4Client">
+        <constructor-arg name="fcrepoBaseURL" ref="fedora4Url" />
```

Comment:
@ajs6f: For the points @mikedurbin mentioned, would you be fine with leaving this as-is, while keeping the "constructor-arg namespace" in mind for future configuration?